.github/workflows/update-plugin-registry.yaml

```diff
@@ -0,0 +1,230 @@
+name: Update Plugin Registry
+
+on:
+  repository_dispatch:
+    types: [plugin-published]
+  workflow_dispatch:
+    inputs:
+      plugin_name:
+        description: "Plugin name (e.g., helminput)"
+        required: true
+        type: string
+      plugin_version:
+        description: "Plugin version (e.g., 0.1.0)"
+        required: true
+        type: string
+      plugin_component:
+        description: "Plugin component name (e.g., ocm.software/plugins/helminput)"
+        required: true
+        type: string
+      dry_run:
+        description: "Perform a dry run without publishing"
+        required: false
+        default: false
+        type: boolean
+
+env:
+  # Repository selection for checkout
+  REGISTRY_CONSTRUCTOR: .github/config/plugin-registry-constructor.yaml
+
+jobs:
+  update-registry:
+    name: Update Plugin Registry
+    runs-on: ubuntu-latest
+    concurrency:
+      group: plugin-registry-publish
+      cancel-in-progress: false  # Queue updates, don't cancel
+    permissions:
+      contents: write
+      packages: write
+      id-token: write
```

Comment:
are you signing the component version for the registry? if not, I dont think you need id-token